{"name": "testcase2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 == 3\n  +2\n  -3", "trace": "@pytest.mark.run(order=2)\n    @allure.title(\"testcase2\")\n    @allure.description(\"I am first test_case2\")\n    @allure.story('story_2')\n    @allure.severity(\"blocker\")\n    def test_case2():\n        print('I am case2')\n        LoggerForApitest.writeInfoLog(\"I AM CASE2\")\n>       assert 2 == 3\nE       assert 2 == 3\nE         +2\nE         -3\n\npytest_case\\test_fixtures1.py:53: AssertionError"}, "description": "I am first test_case2", "attachments": [{"name": "log", "source": "46c0107b-d8b1-4c5d-95e9-46e88be99a0b-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "6b2feb08-6fc4-481e-abfb-0b67bdc3e451-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "8f1739ce-6605-43f1-9eec-3a948b4ba2c0-attachment.txt", "type": "text/plain"}], "start": 1648000790902, "stop": 1648000790902, "uuid": "dbb9f9d5-eabd-4ffd-8713-2f59b1a55b34", "historyId": "92f9d3b642298a60c88279ecf235fa2d", "testCaseId": "725ce7cbcf31b4144284ab868efa6e15", "fullName": "pytest_case.test_fixtures1#test_case2", "labels": [{"name": "story", "value": "story_2"}, {"name": "severity", "value": "blocker"}, {"name": "tag", "value": "run(order=2)"}, {"name": "parentSuite", "value": "pytest_case"}, {"name": "suite", "value": "test_fixtures1"}, {"name": "host", "value": "no-delecs-g42-test-wx405299"}, {"name": "thread", "value": "14820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pytest_case.test_fixtures1"}]}